### PR TITLE
test: assert chart frequency have a unique and default value.

### PR DESCRIPTION
**Pull request by ceoaliongroo**
#338

### CODE REVIEW

File: behat/features/bootstrap/FeatureContext.php [hunk 1]
```diff
@@ -135,6 +134,17 @@ public function iShouldSeeAMarkerSelected($appear = TRUE) {
 
 
   /**
+   * @Then I should have :frequency as chart usage label
+   */
+  public function iShouldHaveAsChartUsageLabel($frequency) {
+    // Because the svg tag i have to use angular.element ti create the assertion
```
Comment (amitaibu) [inline behat/features/bootstrap/FeatureContext.php]: remove comment

File: behat/features/bootstrap/FeatureContext.php [hunk 2]
```diff
@@ -236,4 +246,33 @@ private function waitForXpathNode($xpath, $appear = TRUE) {
     });
   }
 
+  /**
+   * Wait for the text appear or disappear frommthe element in the CSSPath.
```
Comment (amitaibu) [inline behat/features/bootstrap/FeatureContext.php]: fix

File: behat/features/router.feature [hunk 3]
```diff
@@ -10,7 +10,14 @@ Feature: Router
...
-    Given I am an anonymous user
-    When I visit "/"
-    Then I should see the login page
+  Scenario: Set default chartFreq monthly
+    Given I login with user "carlos"
+    When I visit "/#/dashboard/1"
+    Then I should have "חודש" as chart usage label
+
+  @javascript
+  Scenario: Show url with
```
Comment (amitaibu) [inline behat/features/router.feature]: ?

File: client/app/scripts/controllers/menu.js [hunk 4]
```diff
@@ -2,6 +2,8 @@
 
 angular.module('negawattClientApp')
   .controller('MenuCtrl', function($scope, $state, $stateParams, amMoment, Timedate, Category, account, profile) {
+    console.log('leafletDirectiveMarker.click', $stateParams)
```
Comment (amitaibu) [inline client/app/scripts/controllers/menu.js]: remove

**Comment by ceoaliongroo** [comment]
Closed in favor PR #347

**Pull request closed without merging**